rsc/settings.properties

```diff
@@ -26,7 +26,7 @@ LABEXTENDED=yes
 LABMULTIPLEINSERT=yes
 INTERNALPHARMACIES=yes
 MERGEFUNCTION=yes
-SMSENABLED=no
+SMSENABLED=yes
```

Comment:
don't know yet if to enable it by default

Reply:
You are true. Done.